source/marc/construct-enums.rq

```diff
@@ -1242,6 +1242,9 @@ construct {
         (marc:SerialsFrequencyType	marc:BimonthlyIncludes67Or8NumbersAYear	marc:SerialsFrequencyType-b	"b"	UNDEF	UNDEF	"Varannan månad (6-8 ggr/år)"@sv	"Bimonthly Includes 6, 7, or 8 numbers a year."@en)
         (marc:SerialsFrequencyType	marc:ThreeTimesAWeek	marc:SerialsFrequencyType-i	"i"	UNDEF	UNDEF	"Tre gånger i veckan"@sv	"Three times a week"@en)
         (marc:SerialsFrequencyType	marc:MonthlyIncludes91011Or12NumbersAYear	marc:SerialsFrequencyType-m	"m"	UNDEF	UNDEF	"Varje månad (9-12 ggr/år)"@sv	"Monthly Includes 9, 10, 11, or 12 numbers a year."@en)
+        (marc:SerialsFrequencyType	marc:NoDeterminableFrequency	marc:SerialsFrequencyType-_	"_"	UNDEF	UNDEF	"Frekvens kan ej fastställas"@sv	"No determinable frequency, Frequency is intentionally irregular."@en)
```

Comment:
Maybe shorten the English label too (i.e. remove "is intentionally irregular", since it's an implied "or" there and thus of no definitive meaning)?

Reply:
It could be noted in a skos:comment something like "Used when the frequency is known to be intentionally irregular."

Reply:
NEVERMIND :)